discordpp/categories/guild.hh

```diff
@@ -464,8 +464,7 @@ sptr<const std::string> render_target() override {
     }
     return std::make_shared<const std::string>(out);
 }
-#include <discordpp/macros/defineCallClose.hh>*/
-
+#include <discordpp/macros/defineCallClose.hh>
 
 //https://discord.com/developers/docs/resources/guild#get-guild-roles
```

Comment:
I'd like to try to keep stuff in the same order as the API so this goes after the last 2 endpoints

Reply:
I will swap them around then, sorry for that